### DISCUSSION TITLE
Loading the game after selecting a non-base game language setting - fatal error!

### DISCUSSION THREAD

**Discussion by goldenarrow** [voxeltycoon/issues#590] [Modding]
Modded languages can appear in settings, only whilst playing a game. If one is selected, then returns to the main menu, this error occurs:

![image](https://user-images.githubusercontent.com/38834964/92304071-b0382080-ef72-11ea-8c5e-412bc53e5eb1.png)

It also prevents the game being loaded, locking you out and unable to make any changes.

**Steps to reproduce**
Create mod with new language. Create a new game with mod loaded. From that game, change language to modded language. Return to main menu = screenshot as above.

**Build version**
0.84.1

**Platform**
Windows 10

**Reproducibility**
Every time.

**Logs/saves**
[Player.log](https://github.com/voxeltycoon/bug-tracker/files/5177990/Player.log)

**Comment by andrewpey** [comment]
Modding of currencies and names is not supported yet, but we'll add support for them in the future.
